packages/eslint-plugin/docs/rules/lines-between-class-members.mdx

```diff
@@ -21,8 +21,6 @@ In addition to the options supported by the `lines-between-class-members` rule i
   - `"exceptAfterOverload": true` (default) - Skip checking empty lines after overload class members
   - `"exceptAfterOverload": false` - **do not** skip checking empty lines after overload class members
 
-- [See the other options allowed](https://github.com/eslint/eslint/blob/main/docs/rules/lines-between-class-members.md#options)
-
```

Comment:
I removed this link to avoid duplication (+ this link is 404)

<img alt="duplicated link to base rule options" src="https://github.com/typescript-eslint/typescript-eslint/assets/61150013/ba8d8061-369b-469c-91e8-7e5bb3ebc5a4">